src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx

```diff
@@ -70,9 +75,9 @@ class ResponsiveTable extends Component {
     const rows = data.map(this.renderRow);
 
     return (
-      <table className="responsive">
+      <table className="responsive" role="table">
         <thead>
```

Comment:
Do _all_ tables need captions, or at least an `aria-live` region on the page? @jenstrickland?

Reply:
@Mottie I discussed this with @jenstrickland and we decided because of the content above the table we would use `aria-labelledby` instead of a caption and add IDs to the header and content. I added that [here](https://github.com/department-of-veterans-affairs/vets-website/pull/14401/commits/5fa19b2e1dc9361acac84685abccce366c33bbc1)

Reply:
> While they are not required in every case to meet WCAG, captions and summaries are fairly straightforward ways to provide such information that is often needed. A caption functions like a heading for a table. Most screen readers announce the content of captions. ... A summary is usually only needed for complex tables.

So, yeah, a caption is preferred, but on VA.gov there hasn't been a consistent use of them. It would be better if the heading and paragraph that pair with a table were put into the caption, so that a screen reader user would have the context associated with it while using the table rotor. 

@cohnjesse Did you use the `aria-labelledby` to associate the heading above with the table? I don't see an `aria-labelledby` on this table element.